securedrop_client/gui/widgets.py

```diff
@@ -1765,14 +1765,13 @@ def _set_reply_state(self, status: str) -> None:
         elif status == 'PENDING':
             self.setStyleSheet(self.CSS_REPLY_PENDING)
 
-    @pyqtSlot(str)
-    def _on_reply_success(self, message_id: str) -> None:
+    @pyqtSlot(str, str, str)
+    def _on_reply_success(self, source_id: str, message_id: str, content: str) -> None:
         """
         Conditionally update this ReplyWidget's state if and only if the message_id of the emitted
         signal matches the message_id of this widget.
         """
         if message_id == self.message_id:
-            logger.debug('Message {} succeeded'.format(message_id))
```

Comment:
removed this as well as the message debug log since we log almost the exact same thing from the controller